study-builder/studies/rarex/snippets/dob-patient-required.conf

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
This one is used in both lar-consent and lar-assent:

![Screen Shot 2021-05-11 at 11 50 18 AM](https://user-images.githubusercontent.com/5713833/117847598-7e3d3380-b250-11eb-8b41-3712a91b75eb.png)